integration-tests/deployment/keystone/ocr3_deployer.go

```diff
@@ -0,0 +1,46 @@
+package keystone
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/ocr3_capability"
+)
+
+type OCR3Deployer struct {
+	lggr     logger.Logger
+	contract *ocr3_capability.OCR3Capability
+}
+
+var OCR3CapabilityTypeVersion = deployment.TypeAndVersion{
+	Type:    OCR3Capability,
+	Version: deployment.Version1_0_0,
+}
+
+func (c *OCR3Deployer) deploy(req deployRequest) (*deployResponse, error) {
```

Comment:
you could probably follow the same pattern we have with Go generics here https://github.com/smartcontractkit/chainlink/blob/9efabb039f78c1890a4a3749fb4986d6b7947733/integration-tests/deployment/ccip/deploy.go#L92 to avoid unique deploy methods per contract

Reply:
yeah. i thought unique deploy funcs are easier to understand and simpler to start, if more verbose. may revisit